### PR TITLE
Appointment notifications

### PR DESCRIPTION
**Pull request by benlovell**
Sends customer / guider notifications by email and pusher respectively.

For now this is only triggered when the resource manager batch updates appointments. I'll sneak around separately and ensure notifications are triggered in the correct places.

Details are in the commits. I'm going to tidy up the date formatting separately so ignore that for the time being please.

### CODE REVIEW

File: app/views/calendars/show.html.erb [hunk 1]
```diff
@@ -7,4 +7,38 @@
...
+      </div>
+      <div class="modal-body"></div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<% content_for :body_end do %>
```
Comment (AndrewVos) [inline app/views/calendars/show.html.erb]: Can we move this out of the view?

File: config/environments/test.rb [hunk 2]
```diff
@@ -41,6 +41,9 @@
   # Precompile in travis to resolve poltergeist timeouts
   config.assets.compile = ENV['TRAVIS'] ? false : true
 
+  # work off jobs inline
+  config.active_job.queue_adapter = :inline
```
Comment (AndrewVos) [inline config/environments/test.rb]: You can probably delete those `perform_enqueued` called now?

**Comment by AndrewVos** [comment]
This looks good.. Rebase and merge :)

File: config/environments/test.rb (hunk 2 above)
Comment (benlovell) [inline config/environments/test.rb]: Scum bag.

**Pull request merged**